src/app/tests/suites/include/ConstraintsChecker.h

```diff
@@ -299,6 +299,17 @@ class ConstraintsChecker
         return CheckConstraintMinValue(itemName, current.Value(), static_cast<T>(expected));
     }
 
+    template <typename T, typename U, std::enable_if_t<!std::is_pointer<U>::value, int> = 0>
+    bool CheckConstraintMinValue(const char * itemName, const chip::app::DataModel::Nullable<T> & current,
+                                 chip::app::DataModel::Nullable<U> expected)
+    {
+        if (current.IsNull())
```

Comment:
I think this needs to be a check on the expected - the next call will check the current value, but expected could also be null in this case.